presto-main/src/test/java/io/prestosql/execution/TestSqlTask.java

```diff
@@ -114,10 +114,10 @@ public void testEmptyQuery()
                 createInitialEmptyOutputBuffers(PARTITIONED)
                         .withNoMoreBufferIds(),
                 OptionalInt.empty());
-        assertEquals(taskInfo.getTaskStatus().getState(), TaskState.RUNNING);
+        assertTrue(taskInfo.getTaskStatus().getState().isRunning());
```

Comment:
please test actual states and don't use `isRunning()`. We need to have test where we validate `FLUSHING` state